src/rpc/rawtransaction.cpp

```diff
@@ -384,7 +385,8 @@ UniValue gettxoutproof(const UniValue& params, bool fHelp)
     if (!pblockindex)
     {
         CTransaction tx;
-        if (!GetTransaction(oneTxid, tx, consensusParams, hashBlock, false) || hashBlock.IsNull())
+        uint32_t height;
+        if (!GetTransaction(oneTxid, tx, consensusParams, hashBlock, &height, false) || hashBlock.IsNull())

```

Comment:
won't change if you move pnBlockHeight parameter after fAllowSlow

Reply:
fixed, thanks